doc/index.rst

```diff
@@ -1,58 +1,41 @@
+Autocrypt support for mail agents
+=================================
 
-Autocrypt Python Reference tool and implementation
-==================================================
+py-autocrypt provides a :doc:`cmdline tool <cmdline>` and a :doc:`Python API <api>`, to help mail agents evolve `Autocrypt <https://autocrypt.org>`_ support.
 
```

Comment:
Maybe here clarify that intends to implement things that are not autocrpyt goal?